src/sparseml/exporters/transforms/fold_identity_initializers.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+from onnx import ModelProto, NodeProto
+
+from sparseml.exporters.transforms import BaseTransform
+from sparseml.onnx.utils import ONNXGraph, check_load_model, validate_onnx_file
+
+
+def is_identity(node: NodeProto, graph: "ONNXGraph") -> bool:
```

Comment:
One of my goals for OnnxTransform is to have it specified like:

```python
class OnnxTransform(BaseTransform):
    def iter_matches(self, graph: ONNXGraph) -> Iterable[<something>]:
        ...
    def transform_match(self, graph: ONNXGraph, match: <something>):
        ...
    def _transform(self, model):
        for match in self.iter_matches(graph):
             self.transform_match(graph, match)
```

And then in that way we could have FoldIdentityInitializers look like:

```python
class FoldIdentityInitializers(ONNXTransform):
     def iter_matches(self, ...):
          <pull is_identity into this function>
     def transform_match(self, ...):
         < pull inner loop into this function>
```

Reply:
Let's sync on this on Monday, this looks like a good idea.